crates/simulations/src/cozy/analysis/mod.rs

```diff
@@ -1,3 +1 @@
 pub mod base;
```

Comment:
Removed all the old analysis runners, since they relied on the old cozy sim set-ups. Just have a base case, default cozy sim in this PR.